frontend/index.html

```diff
@@ -1,164 +1,421 @@
+
+					<div id="program-menu" class="mr-auto">
+						<button class="custom-toggler navbar-toggler" type="button" data-toggle="collapse" data-target="#topnav-collapse" aria-controls="topnav-collapse" aria-expanded="false" aria-label="Toggle navigation">
+							<span class="navbar-toggler-icon"></span>
+							<span class="navbar-toggle-label">Menu</span>
+						</button>
+					</div>
+
+					<!-- Navigation -->
+					<div class="navbar-collapse collapse" id="topnav-collapse">
+						<ul class="nav navbar-nav">
+							<li data-topnav="metric" class="navbar-text">
+								Metric:
+								<select id="metric" style="width: 100%"></select>
+							</li>
+							<li data-topnav="month" class="navbar-text">
+								Month:
+								<select id="date" style="width: 100%"></select>
+							</li>
+							<li data-topnav="program" class="navbar-text">
+								Program:
+								<select id="program" style="width: 100%"></select>
+							</li>
+							<li class="form-inline">
+								<button class="btn btn-danger btn-block js-add-new">Add</button>
+							</li>
+						</ul>
+					</div>
+
+				</nav>
+			</div><!--container-->
+		</div><!--program-header-->
+		<!-- mobile search button -->
+		<button type="button" class="custom-toggler site-navbar-toggler site-searchbar-toggle d-md-none" data-toggle="collapse" data-target="#searchbar-collapse-mobile" aria-expanded="false" aria-label="Toggle Search">
+			<span class="fa fa-search" aria-hidden="true"></span>
+		</button>   
+		<!-- Start collapsible site search bar -->
+		<div class="site-search-bar">
+			<div class="container">
+				<div class="row">
+					<div class="col">
+						<nav class="navbar navbar-light">
+							<div class="navbar-collapse collapse" id="searchbar-collapse">
+								<form
+									  method="get"
+									  class="site-searchbar-form form-inline ThunderstoneForm"
+									  action="https://search4.nrel.gov/texis/search/" 
+									  id="ThunderstoneSearchForm">
+
+									<input type="hidden" name="pr" value="metanrel" />
+									<div class="form-group">
+											<label for="query" class="sr-only">Search NREL.gov</label>
+											<input
+												   type="search"
+												   id="query" name="query"
+												   maxlength="2047"
+												   placeholder="Search NREL.gov"
+												   class="form-control"
+												   aria-label="Search NREL.gov">
+											<button type="submit" class="btn btn-default">Search</button>
+									</div>
+								</form>
+							</div>
+						</nav>
+					</div>
+				</div>
+			</div>
+		</div>    
+		<!-- End collapsible site search bar -->
+		<!--startindex-->
+
+		<div id="content" class="container-fluid">
+			<div class="gridster">
+				<ul>
+					<li data-row="1" data-col="1" data-sizex="4" data-sizey="4">
+						Number of trips (Nov 2020)
+						<button class='remove'>x</button>
+						<img src='plots/ntrips_mode_confirm_2020_11_prepilot.png' width=90% height=90%>
+					</li>
```

Comment:
we also need to change this to show the plots for the most recent month instead of the hardcoded results for 2020_11 which is way too old